src/App.js

```diff
@@ -1,11 +1,24 @@
+import Input from "./Components/Input";
+import Header from "./Components/Header";
+import Button from "./Components/Button";
 import "./App.css";
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <h1>Todo App React</h1>
-      </header>
+      <Input
+        type="text"
+        className="user-name"
+        placeholder="Please enter user name"
+      />
```

Comment:
there's not much value in creating an `Input` component passing through the same props as you'd give to an `input` element

instead, create e.g. a `UserNameInput` element which implements all the details, so that in the `<App>` component  the code becomes more clean and easier to read, also the `UserNameInput` component will be reusable without duplicating code:

```suggestion
const UserNameInput = () => (
  <input
    type="text"
    className="user-name"
    placeholder="Please enter user name"
  />
);

const App = () => (
  <div>
    <UserNameInput />
    <ChangeUserButton />
    {/* same idea for all components, only pass in props where a component needs to be configured e.g. `onClick` */}
  </div>
);
```